models/topicsModel.js

```diff
@@ -40,3 +39,10 @@ exports.getAllArticles = () => {
       return articles.rows;
     });
 };
+
+exports.getCommentsByArticleId = (article_id) =>{
+    return db.query(`SELECT * FROM comments
```

Comment:
This wont ensure the order of the comments, think about what clauses you could add to your command